docs/gateway-api/gateway-api-v1beta1.md

```diff
@@ -142,6 +148,12 @@ A sample HTTPRoute object is shown below:
 
 The above HTTPRoute object gets translated to two child VS in the AVI controller. One child VS with match criteria as the path begins with `/bar` and a single Pool Group with a single pool and another child VS with match criteria as path begins with `/foo`, a single Pool Group with two pools, and an HTTP Request policy to add `my-header` to the HTTP request forwarded to the backends.
 
+Hostnames are mandatory and cannot contain wildcard.
```

Comment:
All the limitations are under the section `Conditions and Caveats`.

Reply:
it should be fine to have the limitations specified in both the locations